.github/workflows/trigger-release.yml

```diff
@@ -0,0 +1,40 @@
+name: Trigger Release
+
+on:
+  push:
+    branches: [publishWorkflow]
+#  workflow_dispatch:
```

Comment:
I'm going to switch this to `workflow_dispatch` only once we are ready for release